index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse program</title>
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="shortcut icon" href="./images/microverse.png" type="image/x-icon">
+</head>
+<body>
+    <main class="container">
+        <header class="container__header">
+            <nav class="hamburger-menu">
+                <img src="images/hamburger-menu.png" alt="hamburger menu icon">
+            </nav>
+        </header>
+        <section class="introduction">
+            <h1 class="introduction__header">
+                Microverse Full-Stack web development program
+            </h1>
+            <div class="introduction__description-container">
+                <p class="introduction__description">
+                   A program to get you started on Full-Stack web development with Technical and Soft skills required, This  program create positive changes on their alumni, You can join the course wherever you are.
+                </p>
+            </div>
+            <h2 class="event-date">
+                October class: 2021.11.18 (MON) 
+            </h2>
+            
+        </section>
+        
+        <section class="main-program">
+            <h3>
+                Main Program
+            </h3>
+            <div class="main-program__container">
+                <div class="card">
+                    <img src="images/personnel.png" alt="3 men standing in front of a school">
+                    <span class="card-title">Personnel</span>
+                    <p class="card-description">Checkout Microverse staff members </p>
+                </div>
+
+                <div class="card">
+                    <img src="images/course-icon.png" alt="monitor with a book inside it">
+                    <span class="card-title">Curriculum</span>
+                    <p class="card-description">Checkout Microverse program curriculum </p>
+                </div>
+
+                <div class="card">
+                    <img src="images/forum.png" alt="3 men standing in front of a school">
+                    <span class="card-title">Forum</span>
+                    <p class="card-description">Have the time sharing your thoughts and opinion</p>
+                </div>
+
+                <div class="card">
+                    <img src="images/alumni.png" alt="3 men standing in front of a school">
+                    <span class="card-title">Microverse Alumni</span>
+                    <p class="card-description">See Microverse first graduated students opinion on their experience</p>
+                </div>
+                <a href="https://apply.microverse.org/" target="_blank" rel="noopener noreferrer" class="join-btn">
+                Apply now
+                </a>
+            </div>
+        </section>
+    </main>
+</body>
```

Comment:
- [ ] It looks like there's no a footer on index, please add it. 🐱 
![image](https://user-images.githubusercontent.com/20563252/136879453-faae29cb-a1b7-4419-ab2a-6d6f8ba91fc8.png)

- [ ] Please remember:
- Footer
- It contains a logo aligned to the left and text aligned to the right.

Mobile Size
- It has a white background.

Desktop Size
- It has a dark background.